### ISSUE TITLE
Accessibility: Add high contrast mode or tweak text contrast

### ISSUE THREAD

**Issue by jhermsmeier** [resin-io/etcher#1721]
The current text contrast is pretty hard to read for people with impaired vision (and / or lower contrast screens, or unforgiving lighting conditions) – we should think about either adding a high contrast mode to be enabled through the settings, or bumping the overall text contrast.

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2bv1)

**Comment by jhermsmeier** [comment]
See also: https://github.com/electron/electron/pull/7518

**Comment by lurch** [comment]
Probably something for @konmouz to look at?

**Comment by konmouz** [comment]
@jhermsmeier, can you post an example screen shot? The overall contrast ratio should be pretty good at the moment. Is it about the light grey text on the dark grey background?

**Comment by jhermsmeier** [comment]
![wp_20170912_16_40_48_pro](https://user-images.githubusercontent.com/244907/30331942-644261fc-97d9-11e7-983f-d9823f0daab5.jpg)

Simulated by upping the contrast here, half the UI becomes nearly invisible. I'm not sure if there are any good vision impairment simulations we could apply to get a better picture of things that need attention? There's also a description from a user in the linked Front convo, detailing their difficulties.

**Comment by lurch** [comment]
Related to resin-io/hq#1024
